app/src/main/java/org/simple/clinic/widgets/PatientSearchResultView.kt

```diff
@@ -0,0 +1,125 @@
+package org.simple.clinic.widgets
+
+import android.content.Context
+import android.util.AttributeSet
+import android.view.View
+import androidx.cardview.widget.CardView
+import kotlinx.android.synthetic.main.list_patient_search.view.*
+import org.simple.clinic.R
+import org.simple.clinic.activity.TheActivity
+import org.simple.clinic.facility.Facility
+import org.simple.clinic.patient.PatientAddress
+import org.simple.clinic.patient.PatientSearchResult
+import org.simple.clinic.searchresultsview.PhoneNumberObfuscator
+import org.simple.clinic.util.UserClock
+import org.simple.clinic.util.UtcClock
+import org.simple.clinic.util.estimateCurrentAge
+import org.simple.clinic.util.toLocalDateAtZone
+import org.threeten.bp.LocalDate
+import org.threeten.bp.format.DateTimeFormatter
+import javax.inject.Inject
+import javax.inject.Named
+
+class PatientSearchResultView(
+    context: Context,
+    attributeSet: AttributeSet
+) : CardView(context, attributeSet) {
+
+  @field:Inject
+  lateinit var phoneObfuscator: PhoneNumberObfuscator
+
+  @field:[Inject Named("date_for_search_results")]
+  lateinit var dateTimeFormatter: DateTimeFormatter
+
+  @field:Inject
+  lateinit var utcClock: UtcClock
+
+  @field:Inject
+  lateinit var userClock: UserClock
+
+  override fun onFinishInflate() {
+    super.onFinishInflate()
+    TheActivity.component.inject(this)
```

Comment:
`PatientSearchResultView` right now assumes and depends on being used in `list_patient_search.xml`. As you are trying to make this view separate, can you inject the view here? Please refer `PatientSearchView#L53`.

Reply:
Done.